waspc/packages/deploy/src/providers/fly/helpers/CommonOps.ts

```diff
@@ -0,0 +1,42 @@
+import { cdToClientBuildDir, cdToServerBuildDir } from "./helpers.js"
+import * as tomlHelpers from "./tomlFileHelpers.js"
+
+export interface CommonOps {
```

Comment:
It kinda feels a bit specific, but I agree `CommonOps` feels a bit generic. It really represents operations in a specific context (server or client). I don't know if being in one of those contexts is really the "current Wasp project" though 🤔

Reply:
Hm good point. But maybe it doesn't matter that it is client vs server -> yes, they will produce different CmdOptions, but they are still just CmdOptions.

What are these? One cds to project build dir, one checks for toml in project, one copies toml to project, one copies project toml locally. So all of them are doing sometihg with files in the project. WHich makes sense, that is why it has only two fields, waspDir and toml paths. That is why I thought it could be named in that direction. But you think it might grow into something more?

We don't have to over complicate this, if you can find a better name great, if not we can leave it like this.